go.mod

```diff
@@ -45,7 +46,7 @@ require (
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.1.0-rc2.0.20221005185240-3a7f492d3f1b
 	github.com/opencontainers/runc v1.1.5
-	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
+	github.com/opencontainers/runtime-spec v1.1.0-rc.1
```

Comment:
I see that the `1.1.0` spec has been released, so we could cut a `0.6.1` cdi release that bumps this dependency.

Reply:
Does CDI depends on some features of the runtime-spec > 1.0.3? If it doesn't, can we downgrade this and other dependencies to minimise an implact to the Containerd dependencies?

Reply:
CDI does not explicitly depend on a specific CDI spec version.

One issue with using an older spec version could cause new fields to be dropped when loading such as spec from file. Note that in the case of continer-device-interface, we have a number of downstream dependencies which would all determine our target spec version. For example, `podman` and `moby` both use:
```
	github.com/opencontainers/runtime-spec v1.1.0
```
as does the `main` branch of `containerd`.  Would bumping this dependency in the `1.6` branch (and also in our repo) be a better solution?